management/test_lifecycle.py

```diff
@@ -0,0 +1,104 @@
+import sys
+from unittest import mock
+
+import pytest
+
+# avoid requiring PyGitHub to run tests
+sys.modules["github"] = mock.MagicMock()
+
+
+@pytest.mark.parametrize(
+    "input_tag,want",
+    [
+        ("compute-openff_mw-300-600", ([300.0, 600.0], "compute-openff")),
+        (
+            "compute-pr393_mw-300-600-900",
```

Comment:
(not blocking) To make this more readable, maybe replace the `pr393` with something obviously fake.